### PR TITLE
Fix WindowsAI build

### PR DESCRIPTION
**Pull request by tiagoshibata**
**Description**: Changes to WinML to fix WAI build breaks

### CODE REVIEW

File: winml/test/concurrency/ConcurrencyTests.cpp [hunk 1]
```diff
@@ -253,11 +253,11 @@ void MultiThreadMultiSessionOnDevice(const LearningModelDevice& device) {
 }
 
 void MultiThreadMultiSession() {
-    MultiThreadMultiSessionOnDevice(LearningModelDeviceKind::Cpu);
+    MultiThreadMultiSessionOnDevice(LearningModelDevice(LearningModelDeviceKind::Cpu));
```
Comment (tiagoshibata) [inline winml/test/concurrency/ConcurrencyTests.cpp:256]: Not sure why these were needed, in ORT it's calling the correct constructor for `LearningModelDevice(const LearningModelDeviceKind&)`, while in the WAI build it must be given explicitly... Maybe some mismatching compiler options

**Comment by HectorSVC** [comment]
Could you create a separate build pipeline? It mixed with CUDA build piepleline now, and is only covered by code build, is never going to be test covered.

**Comment by snnn** [comment]
> Could you create a separate build pipeline? It mixed with CUDA build piepleline now, and is only covered by code build, is never going to be test covered.

I'd prefer to not having more pipelines. We don't have so many build machines.

**Comment by HectorSVC** [comment]
> > Could you create a separate build pipeline? It mixed with CUDA build piepleline now, and is only covered by code build, is never going to be test covered.
> 
> I'd prefer to not having more pipelines. We don't have so many build machines.
mixed it with CUDA is not a good choice. no test coverage is not acceptable. how about create a nightly build pipeline which is daily base?

**Comment by snnn** [comment]
> mixed it with CUDA is not a good choice. no test coverage is not acceptable. how about create a nightly build pipeline which is daily base?

I feel it is ok. You guys may discuss it with Pranav .

**Comment by tiagoshibata** [comment]
I rebased this branch and resolved conflicts. Jeff applied most fixes to master manually, so the PR is smaller now

**Pull request closed without merging**